phpunit.xml.dist

```diff
@@ -1,15 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false"
-         bootstrap="vendor/autoload.php">
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

Comment:
Referencing schema helps with auto-completion, and referencing the locally installed schema ensures that it's always up-to-date.

Reply:
what if one install only prod-deps without local phpunit and executed global phpunit?

Reply:
Then that person can't make use of the auto-completion feature. However, `phpunit` will ignore attributes it doesn't know of, so no problem if the globally installed `phpunit` is unaware of any configurations.

Reply:
great to hear :+1:
I was afraid of some crash

Reply:
actually, could you propose same for PHP CS Fixer (2.2 branch) ?

Reply:
Will do later!